package.json

```diff
@@ -4,45 +4,43 @@
   "description": "Nest TypeScript starter repository",
   "license": "MIT",
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
```

Comment:
Why remove?

Reply:
The `build` script is still there. The content of `tsconfig.build.json` has been merged with `tsconfig.json`